src/RestQuery.js

```diff
@@ -866,7 +866,7 @@ function includePath(config, auth, response, path, restOptions = {}) {
           return set;
         }
       }
-      if (i == keyPath.length - 1) {
```

Comment:
Is this just code style, or does it have any effect?

Reply:
Either eslint or prettier removes the braces, forcing the tests to fail. So I changed it so that tests pass, and ESLint doesn't break it

Reply:
Wait I'm sure there were braces there before, I can revert if preferred

Reply:
I'd surprised if eslint or prettier make changes that break code, they should only do style changes.

Reply:
I was mistaken